docs/gettinginvolved/infrastructure/vagrant.rst

```diff
@@ -9,21 +9,16 @@ See the `Vagrant documentation <http://docs.vagrantup.com/v2/>`_ for more detail
 Boxes
 -----
 
-There are several vagrant boxes.
+There is currently one vagrant box:
```

Comment:
Actually, we're currently not supporting the Vagrant dev box either (it's broken, we've got no plans to fix it). Maybe best to get rid of both from the docs?

Leaving it up to you.

:question: